remote/rest_rest_windows_test.go

```diff
@@ -1,4 +1,4 @@
-package rest
```

Comment:
File should be just `rest_windows_test.go`

Reply:
I think it is?

Reply:
Right now it's `rest_rest_windows_test.go`.